collector.go

```diff
@@ -6,6 +6,18 @@ import (
 
 //Define the metrics we wish to expose
 var (
+	zcashdBlockchainInfo = prometheus.NewGaugeVec(
+		prometheus.GaugeOpts{
+			Name: "zcash_network",
```

Comment:
looks like we're breaking from a naming pattern here wherein these 'Name' attributes closely follow the go variable names they're assigned to. In this case 'network (name)' is one of the two components (the other being '(number of) blocks'). Is "zcash_network" a better choice of 'Name' attribute here than something like "zcash_blockchain_info"?